integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/dblocation/DBLocationCarbonTableTestCase.scala

```diff
@@ -40,8 +40,9 @@ class DBLocationCarbonTableTestCase extends QueryTest with BeforeAndAfterAll {
 
   }
 
-  override def beforeAll {
+  override def beforeEach {
     sql("drop database if exists carbon cascade")
+    sql("drop database if exists carbon1 cascade")
```

Comment:
This test case file has another database "carbon2", better to do the same operator
Better to  delete "drop database" command in each test case

Reply:
done .